src/main/java/seedu/address/logic/commands/AddCardCommand.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FRONT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_BACK;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.card.Card;
+import seedu.address.model.card.exceptions.DuplicateCardException;
+
+/**
+ * Adds a person to the address book.
+ */
+public class AddCardCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "addc";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a card to the address book. "
+            + "Parameters: "
+            + PREFIX_FRONT + "FRONT "
+            + PREFIX_BACK + "BACK ";
+
+    public static final String MESSAGE_SUCCESS = "New card added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This card already exists in the address book";
```

Comment:
Ideally these variables will have to be changed to match the context of our flash card product